Notebooks/SentinelUtilities/SentinelAnomalyLookup/anomaly_lookup_view_helper.py

```diff
@@ -0,0 +1,18 @@
+"""
+Anomaly Lookup View Helper:
+This module provides helper methods for UI components.
+"""
+
+import os
+import ipywidgets as widgets
+from ipywidgets import Button, GridBox, Layout, ButtonStyle, IntProgress
+
+class AnomalyLookupViewHelper(object): 
+    def define_int_progress_bar():
```

Comment:
Does this need to be a class? Functions can just be module functions. If they are class functions they should have a @staticmethod decorator

Reply:
Will add @staticmethod next iteration